### PR TITLE
static value for autogain

### PR DESCRIPTION
**Pull request by Mlocik97**
Implementation of #587

### CODE REVIEW

**Comment by Mlocik97** [comment]
While I was programming I was looking on other problem with geometric_mean and undesirably I renamed it without noticing it,... I fixed it back

**Comment by wwmm** [comment]
Ok.

**Comment by wwmm** [comment]
Are you being able to compile PE sources and run it? I ask because the circleci test failed to compile.

Usually I do the following inside PE source folder:

```
- meson _build
- cd _build
- ninja
- cd src
- source ../../util/environmental_variables.sh
- ./pulseeffects
```

**Comment by Mlocik97** [comment]
no, I'm not able to compile it, I added You to access fork of my repo.

**Comment by Mlocik97** [comment]
still need fix saving setting to preset.

**Comment by Mlocik97** [comment]
Can You please create new branch, where should I lead changes to squash to one commit?

**Comment by Mlocik97** [comment]
ok, I think, everything is ok.

**Comment by wwmm** [comment]
I can take care of the gtk interface. But I am not sure when yet. I am involved in a project here in Brazil that is trying to find solutions to help hospitals to fix broken ventilators and I have to prioritize it now. But someday I will work on this pull request :-)

File: src/autogain/gstpeautogain.cpp [hunk 1]
```diff
@@ -439,12 +467,23 @@ static void gst_peautogain_process(GstPeautogain* peautogain, GstBuffer* buffer)
...
+          peautogain->loudness = std::cbrt(peautogain->momentary * peautogain->shortterm * peautogain->global);
+        }
       } else {
-        peautogain->loudness =
-            (peautogain->weight_m * peautogain->momentary + peautogain->weight_s * peautogain->shortterm +
-             peautogain->weight_i * peautogain->global) /
-            (peautogain->weight_m + peautogain->weight_s + peautogain->weight_i);
+        if (peautogain->use_static_integrated_value) {
+          peautogain->loudness =
+              (peautogain->weight_m * peautogain->momentary + peautogain->weight_s * peautogain->shortterm +
```
Comment (Mlocik97) [inline src/autogain/gstpeautogain.cpp:478]: maybe refactor? We can just replace integrated with static value by passing it to parameter of function, so we don't need 2x nearly same code.

File: src/autogain/gstpeautogain.cpp [hunk 2]
```diff
@@ -439,12 +467,23 @@ static void gst_peautogain_process(GstPeautogain* peautogain, GstBuffer* buffer)
...
+               peautogain->weight_i * peautogain->staticv) /
+              (peautogain->weight_m + peautogain->weight_s + peautogain->weight_i);
+        } else {
+          peautogain->loudness =
+              (peautogain->weight_m * peautogain->momentary + peautogain->weight_s * peautogain->shortterm +
+               peautogain->weight_i * peautogain->global) /
+              (peautogain->weight_m + peautogain->weight_s + peautogain->weight_i);
+        }
```
Comment (Mlocik97) [inline src/autogain/gstpeautogain.cpp:486]: Should refactor, so there is no 2x nearly same code... we can extract it to another function and just replace parameter by value we need.

**Comment by Mlocik97** [comment]
I will clean that mess in commits...

**Comment by Mlocik97** [comment]
I totally fucked/screwed up this PR...

**Comment by wwmm** [comment]
Don't  worry. I will keep that copy :-)

**Pull request closed without merging**